### PR TITLE
doc(reports) improve date_range widget documentation

### PR DESCRIPTION
**Pull request by cpuy**
Add more specific details for date_range widget initial values

Closes [BS-15720](https://bonitasoft.atlassian.net/browse/BS-15720)

Once it is merged, `7.4` and `7.5` branches should be rebased onto `7.3` or `7.3` should be merged in those branches...

### CODE REVIEW

File: md/create-a-report.md [hunk 1]
```diff
@@ -165,15 +165,22 @@ The **DATE** and **DATE\_RANGE** widgets have the following extra properties:
...
+        Example: <span><code>2015-01-28 - 2015-02-28</code></span>
+    </td>
   </tr>
   <tr>
     <td>BONITA_FORM_<em>XXX</em>_QUERY</td>
     <td>Query that dynamically retrieves the initial value from the Bonita BPM database.</td>
   </tr>
 </tbody></table>
 
+For a **DATE\_RANGE** widget, if you name the associated parameter `p_date`, `p_date_from` and `p_date_to` initial
```
Comment (abirembaut) [inline md/create-a-report.md]: I would just a add a "then" to ease the understanding of the sentence (, then `p_date_from` and `p_date_to`...).

**Review by abirembaut — CHANGES_REQUESTED** [review]
LGTM (just one small note)

File: md/create-a-report.md (hunk 1 above)
Comment (cpuy) [reply md/create-a-report.md]: ok thx

**Comment by cpuy** [comment]
Done

**Comment by cpuy** [comment]
Up! Up!

**Review by abirembaut — APPROVED** [review]

**Pull request merged**